### ISSUE TITLE
Turbine not producing electricity 1.19.2

### ISSUE THREAD

**Issue by TrueDubzy** [BiggerSeries/BiggerReactors#72]
im playing on all the mods 8 and cannot get the turbine to produce power,

![2023-02-09_20 48 27](https://user-images.githubusercontent.com/20246342/217935978-310f244c-5888-4b0a-b88f-e3ff7421cebb.png)

**Comment by gizmocodes** [comment]
It's difficult to tell exactly whats happening without more information but I think the main thing here is the turbine is spinning _way_ too fast.

Turbines are most efficient at 1800 RPM (iirc), and lose efficiency if you under- or over-speed them. A little after 5000 RPM is when they are moving too fast and stop producing power, so you need to reduce the speed of the turbine. Easiest way is to reduce the input rate of the turbine (if you have plenty of excess steam, consider building multiple turbines to distribute the steam towards). 

Other thing since it's hard to see, but does the turbine have any coil blocks in it? I imagine they're probably hidden by JEI or off the right of the image, but you'll need at least one ring of them for the turbine to produce anything even if it's in the right speed range.

**Comment by TrueDubzy** [comment]
yes, you were correct. i dint look hard enough sorry for wasting your time
